### PR TITLE
Added LambdaTest CircleCI Orb

### PR DESCRIPTION
**Pull request by jahmed2345**
# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

**Comment by jahmed2345** [comment]
Added brief details about LambdaTest

File: jekyll/_cci2/browser-testing.md [hunk 1]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+## LambdaTest
+
+LambdaTest now integrates with CircleCI to boost your go-to-market delivery. Perform automated cross browser testing with LambdaTest to ensure your development code renders seamlessly through an online Selenium grid providing 2000+ real browsers running through machines, on the cloud. Perform automation testing in parallel with LambdaTest’s Selenium grid to drastically trim down your test cycles.
+
+You can analyse detailed test reports of your automation scripts including network logs, command logs, Selenium logs, step-by-step screenshots for every command, entire video of your test execution, metadata, and more.!
+
+To integrate CircleCI with LambdaTest you need to make minor tweaks in the .circleci/config.yml which is the configuration file for your CircleCI instance. The changes you need to make will revolve around the environment variables such as access key, username, grid config and so on. 
+
+Below is a sample config file for integrating CircleCI with LambdaTest.
+
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
{% raw %}
```

File: jekyll/_cci2/browser-testing.md [hunk 2]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+          command: echo ${LT_USERNAME}      
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+      - run: npm install
+      # run tests!
+      - run: node_modules/.bin/nightwatch -e chrome // Executing test in bash.
+ ```
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
 ```
{% endraw %}
```

File: jekyll/_cci2/browser-testing.md [hunk 3]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+ ### Testing Locally Hosted or Privately Hosted Projects
+ 
+ To help you perform cross browser testing of your locally stored web pages, LambdaTest provides an SSH(Secure Shell) tunnel connection with the name Lambda Tunnel. With Lambda Tunnel, you can execute a test server inside your CircleCI build container to perform automated cross browser testing on browsers offered by Selenium Grid on LambdaTest. That way, you could realize how fantastic your website would look in front of your audience, even before you make it live!
+
+The below example of config.yml file would demonstrate you on how to leverage LambdaTest’s Selenium Grid by performing a browser test through your testing server contained in your CircleCI build.
+
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
{% raw %}
```

File: jekyll/_cci2/browser-testing.md [hunk 4]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+   	
+        - run: npm install
+#      - save_cache:
+#        paths:
+#      - node_modules
+#        key: v1-dependencies-{{ checksum "package.json" }}
+  
+    # run tests!
+      - run: node_modules/.bin/nightwatch -e chrome
+```
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
```
{% endraw %}
```

File: jekyll/_cci2/browser-testing.md [hunk 5]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+      - run: npm install
+      # run tests!
+      - run: node_modules/.bin/nightwatch -e chrome // Executing test in bash.
+ ```
+ ### Testing Locally Hosted or Privately Hosted Projects
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
{% raw %}
 ### Testing Locally Hosted or Privately Hosted Projects
```

File: jekyll/_cci2/browser-testing.md [hunk 6]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+        - run: npm install
+#      - save_cache:
+#        paths:
+#      - node_modules
+#        key: v1-dependencies-{{ checksum "package.json" }}
+  
+    # run tests!
+      - run: node_modules/.bin/nightwatch -e chrome
+```
+### LambdaTest Browser Testing Orb Example
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
{% endraw %}
### LambdaTest Browser Testing Orb Example
```

File: jekyll/_cci2/browser-testing.md [hunk 7]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+LambdaTest has developed a CircleCI orb for browser compatibility testing that enables you to open a Lambda Tunnel before performing any browser testing. Also, to ease the process of integration between LambdaTest & CircleCI. This orb (a package of configurations that you can use in your workflow) has been developed and certified for use and can simplify your configuration workflows. An example of the orb is shown below.
+
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
{% raw %}
```

File: jekyll/_cci2/browser-testing.md [hunk 8]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+      - image: <<parameters.docker>>
+    steps:
+      - checkout
+      - install
+      - open_tunnel:
+          tunnel_name: <<parameters.tunnel_name>>
+      - steps: << parameters.steps >>
+      - close_tunnel
+```
 ## Sauce Labs
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
{% endraw %}
## Sauce Labs
```

File: jekyll/_cci2/browser-testing.md [hunk 9]
```diff
@@ -54,8 +54,193 @@ For more information about working with Headless Chrome,
...
+As an alternative to configuring your environment for Selenium, you could move to cloud-based platforms such as LambdaTest, Sauce Labs, or BrowserStack. These cross browser testing clouds provide you with a readymade infrastructure on the cloud so you don’t have to waste time on configuring a Selenium environment on your own. 
 
+## LambdaTest
+
+LambdaTest now integrates with CircleCI to boost your go-to-market delivery. Perform automated cross browser testing with LambdaTest to ensure your development code renders seamlessly through an online Selenium grid providing 2000+ real browsers running through machines, on the cloud. Perform automation testing in parallel with LambdaTest’s Selenium grid to drastically trim down your test cycles.
+
+You can analyse detailed test reports of your automation scripts including network logs, command logs, Selenium logs, step-by-step screenshots for every command, entire video of your test execution, metadata, and more.!
```
Comment (michelle-luna) [inline jekyll/_cci2/browser-testing.md]:
```suggestion
You can analyse detailed test reports of your automation scripts including network logs, command logs, Selenium logs, step-by-step screenshots for every command, entire video of your test execution, metadata, and more.
```

**Pull request merged**